packages/std/src/results/cosmos_msg.rs

```diff
@@ -46,6 +46,7 @@ pub enum CosmosMsg<T = Empty> {
     Wasm(WasmMsg),
     #[cfg(feature = "stargate")]
     Gov(GovMsg),
+    LastMsgMark(T),
```

Comment:
Also, I think `FinilizeTx` is a better name here, as it's not just a "last message marker" within the tx, but also within that last message it's the last thing that will happen.

Reply:
I already merged the changes to the main PR, so I'll change this in a separate one